auth/login.go

```diff
@@ -132,11 +131,6 @@ func newAPIConfig(apiURL, issuerURL *url.URL, command, clientID string, opts ...
 		return clientConfig, nil
 	}
 
-	// we make sure our command is in the $PATH as the client-go credential plugin will need to find it.
-	if _, err := exec.LookPath(command); err != nil && command != "" {
```

Comment:
AFAIK this change only applies to the `nctl auth cluster` command, not `nctl auth login`. As `auth login` also adds an entry to the kubeconfig with the command `nctl`, we still need to ensure `nctl` is in the path or we need to also change `auth login`.

Reply:
I changed `auth login` to also use the same mechanism.